<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-color:qlineargradient(spread:pad, x1:0.49, y1:0, x2:0.502, y2:1, stop:0 rgba(61, 55, 52, 255), stop:0.664145 rgba(47, 42, 39, 255))
}</string>
   </property>
   <widget class="QLabel" name="Date">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#Date{
border-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>821</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#background{
border-radius:10px;
border-width:3px;
border-color: rgb(120, 106, 93);
border-style: solid;
background-color: rgb(43, 39, 36);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bp_back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>401</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#bp_back{
border-radius:10px;
background-color:qlineargradient(spread:pad, x1:0.522727, y1:0, x2:0.529, y2:1, stop:0 rgba(56, 51, 45, 255), stop:0.732955 rgba(34, 30, 26, 255));
border-width:2px;
border-color: rgb(56, 51, 45);
border-style: solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bp_front">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#bp_front{
border-radius:10px;
background-color:  rgb(79, 70, 61);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="item">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #FFD700;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="type">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #756B5E;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="neutral">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #EBE2CA;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="negative">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #FF4040;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="positive">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #99CCFF;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="attrs" native="true">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>350</y>
      <width>361</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#attrs{
background-color: transparent;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>361</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#info{
border-radius:10px;
background-color:qlineargradient(spread:pad, x1:0.522727, y1:0, x2:0.529, y2:1, stop:0 rgba(56, 51, 45, 255), stop:0.732955 rgba(34, 30, 26, 255));
border-width:2px;
border-color: rgb(56, 51, 45);
border-style: solid;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="changelog">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #EBE2CA;</string>
    </property>
    <property name="text">
     <string>Changelog for</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="changelog_date">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #EBE2CA;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="changes" native="true">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>180</y>
      <width>321</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#changes{
background-color: transparent;
}</string>
    </property>
    <widget class="QLabel" name="changes_text">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: #EBE2CA;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="dummy">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>610</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dummy_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>680</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>654</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#back{
border-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt; Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>120</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Skip to next update (Hotkey: Right Arrow)</string>
    </property>
    <property name="toolTipDuration">
     <number>2500</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#next{
border-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);
}</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Skip to previous update (Hotkey: Left Arrow)</string>
    </property>
    <property name="toolTipDuration">
     <number>2500</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#previous{
border-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);
}</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
